### PR TITLE
Fix building for monorepo

### PR DESCRIPTION
**Pull request by iBat**
Converting to monorepo

- PR 1/2: #25096
- PR 2/2: this one

### CODE REVIEW

**Review by pomahtri — DISMISSED** [review]

File: .github/workflows/devextreme_npm_tests.yml [hunk 1]
```diff
@@ -153,21 +149,21 @@ jobs:
 
     - uses: ./.github/actions/pin-browsers
 
-    - name: Create directory link
-      run: cd ../ && ln -s DevExtreme devextreme
+#    - name: Create directory link
```
Comment (IlyaKhD) [inline .github/workflows/devextreme_npm_tests.yml]: Can we remove commented lines?

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -152,37 +160,42 @@ jobs:
...
-
-    - name: Check package-lock
-      run: |
-        node -v
-        npm -v
-        npm install --no-audit --no-fund
-
-    - name: Check build
-      run: npm run build:dev
+# TODO discuss - do we need it? If yes - we should fix it (https://github.com/DevExpress/DevExtreme/actions/runs/5532662586/jobs/10095078667?pr=25082)
```
Comment (IlyaKhD) [inline .github/workflows/lint.yml]: Can we remove commented lines (2)?

File: packages/devextreme/build/pre-commit/index.js.tmp [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict';
```
Comment (IlyaKhD) [inline packages/devextreme/build/pre-commit/index.js.tmp]: What is this file for?

File: packages/devextreme/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "devextreme",
+  "name": "devextreme-root",
```
Comment (IlyaKhD) [inline packages/devextreme/package.json]: I would avoid using 'root' in this case, it may be confusing as this is definitely not a root

Comment (IlyaKhD) [reply packages/devextreme/package.json]: What about `devextreme-main`?

File: packages/devextreme/testing/tests/DevExpress.viz.vectorMap.utils/tests.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 /* global ROOT_URL */
 
-import { parse } from '../../../../artifacts/js/vectormap-utils/dx.vectormaputils.js';
+// TODO fix me! This used to work because of a dirty hack - the application rested on the root.
```
Comment (IlyaKhD) [inline packages/devextreme/testing/tests/DevExpress.viz.vectorMap.utils/tests.js]: Can this comment be deleted (3)?

**Review by IlyaKhD — DISMISSED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Review by alexslavr — APPROVED** [review]

**Pull request merged**